go.mod

```diff
@@ -95,7 +95,7 @@ require (
 	github.com/tdakkota/asciicheck v0.2.0
 	github.com/tiancaiamao/appdash v0.0.0-20181126055449-889f96f722a2
 	github.com/tikv/client-go/v2 v2.0.8-0.20230512034316-adb48afeef3e
-	github.com/tikv/pd/client v0.0.0-20230419153320-f1d1a80feb95
+	github.com/tikv/pd/client v0.0.0-20230519121736-d15a686a670e
```

Comment:
update gomod for fixed https://github.com/tikv/pd/pull/6496
@nolouch PTAL, thx!